app/api/helpers/tasks.py

```diff
@@ -183,15 +183,21 @@ def export_event_task(self, email, event_id, settings):
         result = {
             'download_url': download_url
         }
+
         logging.info('Exporting done.. sending email')
-        send_export_mail(email=email, event_name=event.name, download_url=download_url)
+        if check_smtp_config:
```

Comment:
Isn't this a function?

Reply:
@iamareebjamal Yeah! Missed that. Updated it.